### ISSUE TITLE
origin param in afterLoad call back is null

### ISSUE THREAD

**Issue by subdance** [alvarotrigo/fullPage.js#3671]
in my script
```javascript
afterLoad: this.afterLoad
```
```javascript
methods: {
    afterLoad(origin, destination, direction) {
      console.log(origin)
    }
}
```
then console print out `null`. So how can I get the origin param as your document?

**Comment by subdance** [comment]
Maybe. A `null` in console sometimes makes people confused.

**Comment by alvarotrigo** [comment]
Fixed on the dev branch!
It will get merged soon in the next release 3.0.6!

**Comment by alvarotrigo** [comment]
Merged into fullpage.js 3.0.6 which is now available! ⚡️ ⚡️
